books/free-programming-books-subjects.md

```diff
@@ -666,6 +666,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [How to write a simple operating system in assembly language](http://mikeos.sourceforge.net/write-your-own-os.html) - Mike Saunders (HTML)
 * [Linux From Scratch](https://www.linuxfromscratch.org/lfs/view/stable/) - Gerard Beekmans, Bruce Dubbs, Ken Moffat, Pierre Labastie, et al. (HTML, [PDF, downloads...](https://www.linuxfromscratch.org/lfs/downloads/stable/))
 * [Making Servers Work: A Practical Guide to Linux System Administration](https://www.digitalocean.com/community/books/sysadmin-ebook-making-servers-work) - Jamon Camisso (PDF, EPUB)
+* [Operating System Concepts](https://os.ecci.ucr.ac.cr/slides/Abraham-Silberschatz-Operating-System-Concepts-10th-2018.pdf) - Abraham Silberschatz, Peter Baer Galvin, Greg Gayne
```

Comment:
Is this an authorized distribution? Opening https://os.ecci.ucr.ac.cr,  i see a blank image

![image](https://user-images.githubusercontent.com/50091666/196710011-8289389c-2b05-41b4-9b1b-70167a8afacf.png)

Reply:
The link (with everything including after the "ac.cr") leads to the document. 

If you try previewing the file through the exact link, it should work.